engine/menus/start_menu.asm

```diff
@@ -40,6 +42,9 @@ StartMenu::
 	jr .Select
 
 .Reopen:
+	call RestoreSprites
```

Comment:
I'm wondering how much a performance impact additions like this have on actions like "opening the Start menu". I see that `ClearSpritesUnderStartMenu` unconditionally does its stuff. Can/should these sort of new functions have an initial "if weather isn't happening, just return" case? Or would there be desync issues with weather having just stopped/started but sprites could potentially still be on-screen?

Reply:
We are now backing up and restoring not only weather sprites, but other sprites as well. The performance impact should be negligible from a user perspective.